Jenkinsfile

```diff
@@ -1,8 +1,8 @@
-def recentLTS = "2.176.4"
 def configurations = [
     [ platform: "linux", jdk: "8", jenkins: null ],
-    [ platform: "windows", jdk: "8", jenkins: recentLTS, javaLevel: "8" ],
-    [ platform: "linux", jdk: "11", jenkins: recentLTS, javaLevel: "8" ],
+    [ platform: "windows", jdk: "8", jenkins: null ],
```

Comment:
we don't normally test windows on both 8 and 11, as normally incompatibility between versions is a java thing and not a platform and we save CI time by not doing this
```suggestion
```